spec/controllers/index_spec.rb

```diff
@@ -0,0 +1,78 @@
+require_relative '../spec_helper'
+
+describe 'Index Controller' do
+let (:new_sealion) {Sealion.create}
+
+  it 'Should get the / route' do
+    get '/'
+    expect(last_response).to be_ok
+    expect(last_response.body).to eq('Eveanandi Blossom Butler')
+  end
+
+  it 'Should get the /sealions route' do
+    get '/sealions'
+    expect(last_response).to be_ok
+    expect(last_response.body).to eq('This route lists all of the sealions')
+  end
+
+
+  it 'Should get the /sealions/:id route' do
+    get '/sealions/:id', :params => {id: 5}
```

Comment:
@ebutler90 this could be the reason why you are seeing issues.  What is the actual route here? Shouldn't look something like '/sealions/some-sea-lion-id', :params => {name: 'hi', age: 23}